src/span_context.js

```diff
@@ -68,7 +68,11 @@ export default class SpanContext {
 
   get traceId(): any {
     if (this._traceId == null && this._traceIdStr != null) {
-      this._traceId = Utils.encodeInt64(this._traceIdStr);
+      const safeTraceIdStr = this._traceIdStr.length % 2 == 0 ? this._traceIdStr : '0' + this._traceIdStr;
```

Comment:
What is the objective of checking for even string length? Please add a comment when this could be an issue.

Reply:
In node hex strings have to be of [even length](https://github.com/nodejs/node/issues/21242) for Buffer.From(), I'll add a comment